@here/harp-omv-datasource/lib/OmvRestClient.ts

```diff
@@ -216,19 +216,29 @@ export interface OmvRestClientParameters {
      * @deprecated Please use [[authenticationCode]].
      */
     getBearerToken?: () => Promise<string>;
+
+    /**
+     * Array of query parameters to be appended at the end of the url.
+     * It is empty by default.
+     */
+    urlParams?: Array<[string, string]>;
```

Comment:
why array of arrays? Would be more natural to write sth. like:
```
{
    "param": "value",
    ...
}
```

Reply:
I just took the same value as in the private function, but agree, that's nicer, let me change it